pep-0570.rst

```diff
@@ -149,13 +203,12 @@ Providing positional-only arguments will allow for maintaining the
 interface when creating pure Python implementation of C modules, which
 provides not only the API benefits outlined in this document but it is
 also faster. See this thread about converting keyword arguments to positional:
```

Comment:
This sentence has a lot packed into it and I think it ends up being a bit of a conceptual jumble, to untangle, I think that there are three semi-unrelated points being made here:

1. Having some arguments be positional-only is valuable, because it has the API benefits described in the earlier section.
2. Functions in extension modules use positional-only modules, and it is difficult or impossible to properly maintain a pure Python implementation of these functions at the moment.
3. Positional-only arguments are faster in extension modules, and when we have proper syntax for them, they'll *definitely* be faster in pure Python than a corresponding `*args`-based implementation would be.

I think point 2 explains why, *given that we have positional-only arguments*, we should have a syntax to specify them in function definitions, while points 1 and 3 explain why it's a good thing that we have positional-only arguments. A potential rewording for this whole paragraph:

"In addition to the API benefits outlined earlier in this document, positional-only arguments are also faster, as demonstrated in this thread about converting keyword arguments to positional: [#thread-keyword-to-positional]_. Providing a way to specify positional-only arguments in Python will make it easier to maintain pure Python implementations of C modules and will allow users to take advantage of these benefits even in code written only in Python."